benchmarks/dynamo/distributed.py

```diff
@@ -81,6 +83,9 @@ def move_tensor(maybe_tensor):
             dynamo.config.log_level = logging.DEBUG
         if args.dynamo_optimize_ddp:
             dynamo.config.optimize_ddp = True
+        if args.dynamo == "inductor" and args.fsdp:
+            torch._inductor.config.triton.cudagraphs = False
+            log.warn("disabling inductor cudagraphs for compatibility with FSDP")
```

Comment:
is this related or something different?

Reply:
well, it's somewhat coupled.  There were weird bugs when I ran with cudagraphs, and i haven't debugged yet.  I think we agreed to make cudagraphs disabled by default in inductor, which would make this unnecessary.